AudiusProject/audius-protocol
# Add weather map

### Description
Adds weather map to inspect buckets, search files, detect inconsistencies, and more! As a side task, this also renames bucket to shard across Storage Node (would've done separately but didn't want to make the whole weather map use outdated names).
<img width="1443" alt="Screen Shot 2023-02-07 at 10 31 00 AM" src="https://user-images.githubusercontent.com/4657956/217334181-bc809417-68d1-4584-93af-a8d8b5d9914a.png">



### Tests
- `docker compose -f docker-compose.v2.yml up -d --build`
- http://localhost:9924/storage/weather?demo=true
- Play around and see if there are any issues
  - Upload content at `/storage` to make sure you can search it by fileName and jobID (note that the fake data, including the "DEMO" shard, aren't fully searchable)
  - View the demo shard to see what inconsistency highlighting looks like - http://localhost:9924/storage/weather/shard/DEMO


### Monitoring - How will this change be monitored? Are there sufficient logs / alerts?
N/A
